python/ray/worker.py

```diff
@@ -1224,11 +1246,11 @@ def _initialize_serialization(worker=global_worker):
     This defines a custom serializer for object IDs and also tells ray to
     serialize several exception classes that we define for error handling.
     """
-    worker.serialization_context = pyarrow.default_serialization_context()
+    worker.serialization_context_map[worker.task_driver_id] = pyarrow.default_serialization_context()
```

Comment:
current_context = pyarrow.default_serialization_context()
and set to worker.serialization_context_map[worker.task_driver_id] in the end of the function.